orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/RestrictExecutionDuringTimeWindow.groovy

```diff
@@ -41,8 +44,28 @@ class RestrictExecutionDuringTimeWindow implements StageDefinitionBuilder {
   public static final String TYPE = "restrictExecutionDuringTimeWindow"
 
   @Override
+  @CompileStatic(TypeCheckingMode.SKIP)
   void taskGraph(Stage stage, TaskNode.Builder builder) {
     builder.withTask("suspendExecutionDuringTimeWindow", SuspendExecutionDuringTimeWindowTask)
+    if (stage.context.restrictedExecutionWindow?.splay?.enabled && stage.context.restrictedExecutionWindow?.splay?.maxDelay) {
```

Comment:
My english is pretty weak but `splay` is a new one to me.

Reply:
https://www.merriam-webster.com/dictionary/splay

Reply:
I'd be more ok if we called this `jitter` instead.

Reply:
Aside from that ... might be a tad cleaner if you defined a new object and did stage.mapTo("/restrictedExecutionWindow/jitter", JitterConfig).

Reply:
stage.mapTo sounds better, will do.

`splay` is actually fairly common in infrastructure automation tooling, used in saltstack, puppet, chef, and cfengine for this purpose.

https://puppet.com/docs/puppet/5.3/configuration.html#splay
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.schedule.html
https://docs.cfengine.com/docs/3.5/reference-components-cfexecd.html#splaytime
https://docs.chef.io/config_rb_client.html